pkg/cstor/volume/v1alpha1/cstorvolume.go

```diff
@@ -37,6 +45,20 @@ type ListBuilder struct {
 	filters PredicateList
 }
 
+// GetResizeConditions will return reaize condtions related to
```

Comment:
spelling mistake for `resize`..
why plural form of GetResizeConditions? can this be singular?

Reply:
Initially thought of having multiple resize conditions ex: Resizing, Resized... and later thought of one alone is enough. Will update the `func` name.

Reply:
>  why plural form of GetResizeConditions? can this be singular?

Yes